shell/src/main/java/alluxio/cli/fsadmin/report/ConfigurationCommand.java

```diff
@@ -24,6 +24,7 @@
  * Prints runtime configuration information.
  */
 public class ConfigurationCommand {
+  private static final String CONFIG_INFO_FORMAT = "%s (value = %s, source = %s)%n";
```

Comment:
Now we print all kinds of configurations in the report CLI, some of the properties have long value. The previous way of defining the column wide as the length of longest string belongs to that column seems to be improper. As a result, I change to the current format.

Reply:
Good idea, it's much better this way